generator/test/generator_test.dart

```diff
@@ -886,51 +886,6 @@ class _Bad1 {}
     );
   });
 
-  test('defining both _Bad and \$Bad in different files', () async {
```

Comment:
Oh yes, this also allows defining similar named models in multiple files, as discussed on round-table

Reply:
Moved to separate PR